<rss xmlns:atom="http://www.w3.org/2005/Atom" version="2.0">
    <channel>
        <title>Celery - 标签 - cywhat&#39;s blog</title>
        <link>https://cywhat.cn/tags/Celery/</link>
        <description>Celery - 标签 - cywhat&#39;s blog</description>
        <generator>Hugo -- gohugo.io</generator><language>zh-CN</language><managingEditor>cy-support@cywhat.cn (cywhat)</managingEditor>
            <webMaster>cy-support@cywhat.cn (cywhat)</webMaster><copyright>This work is licensed under a Creative Commons Attribution-NonCommercial 4.0 International License.</copyright><lastBuildDate>Tue, 18 Apr 2023 14:42:49 &#43;0800</lastBuildDate><atom:link href="https://cywhat.cn/tags/Celery/" rel="self" type="application/rss+xml" /><item>
    <title>Django操作异步任务</title>
    <link>https://cywhat.cn/Django%E6%93%8D%E4%BD%9C%E5%BC%82%E6%AD%A5%E4%BB%BB%E5%8A%A1/</link>
    <pubDate>Tue, 18 Apr 2023 14:42:49 &#43;0800</pubDate>
    <author>作者</author>
    <guid>https://cywhat.cn/Django%E6%93%8D%E4%BD%9C%E5%BC%82%E6%AD%A5%E4%BB%BB%E5%8A%A1/</guid>
    <description><![CDATA[前置条件 1 2 3 4 5 Python==3.7.0 Pip==3 Django==3.2.18 celery==5.2.7 redis==3.5.3 1、安装 1 pip3 install celery 2、目录 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 - Heng_Tools/ - manage.py - Heng_Tools/ - __init__.py # 修改这个文件 - celery.py # 新增这个文件 - asgi.py]]></description>
</item><item>
    <title>ValueError:invalid</title>
    <link>https://cywhat.cn/ValueErrorinvalid-literal-for-intwithbase10-63790/</link>
    <pubDate>Fri, 14 Apr 2023 15:51:46 &#43;0800</pubDate>
    <author>作者</author>
    <guid>https://cywhat.cn/ValueErrorinvalid-literal-for-intwithbase10-63790/</guid>
    <description><![CDATA[1、前景 使用django的异步调用场景时，并配置了如下配置：出现了错误：ValueError: Database is int between 0 and limit - 1, not :6379/0 1 2 #CELERY_BROKER_URL =&#34;redis://127.0.0.1/:6379/0&#34; #CELERY_BACKEND_URL = &#34;redis://127.0.0.1/:6379/1&#34; 2、原因 1]]></description>
</item></channel>
</rss>
